src/v2/Apps/Home/Components/HomeAuctionLotsRail.tsx

```diff
@@ -0,0 +1,162 @@
+import {
+  Box,
+  Text,
+  Flex,
+  Spacer,
+  Shelf,
+  Skeleton,
+  SkeletonText,
+  SkeletonBox,
+  Sup,
+} from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "v2/System"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { HomeAuctionLotsRail_viewer } from "v2/__generated__/HomeAuctionLotsRail_viewer.graphql"
+import { HomeAuctionLotsRailQuery } from "v2/__generated__/HomeAuctionLotsRailQuery.graphql"
+import { extractNodes } from "v2/Utils/extractNodes"
+import { ShelfArtworkFragmentContainer } from "v2/Components/Artwork/ShelfArtwork"
+
+interface HomeAuctionLotsRailProps {
+  viewer: HomeAuctionLotsRail_viewer
+}
+
+const HomeAuctionLotsRail: React.FC<HomeAuctionLotsRailProps> = ({
+  viewer,
+}) => {
+  const nodes = extractNodes(viewer.saleArtworksConnection).filter(node => {
+    return !node.sale?.isClosed
+  })
+
+  if (nodes.length === 0) {
+    return null
+  }
+
+  return (
+    <HomeAuctionLotsRailContainer lotCount={nodes.length}>
+      <Shelf>
+        {nodes.map((node, index) => {
+          return (
+            <Box>
+              <ShelfArtworkFragmentContainer
+                artwork={node}
+                key={node.slug}
+                // TODO: Add home type to cohesion once we have tracking
+                contextModule={null as any}
+                hidePartnerName
+                // FIXME: For some reason this is leaves the gray loading
+                // background post load
```

Comment:
That's always there (it's on my grievances list). But why is it a problem in this case? It should only matter for transparent images.

Reply:
Wrong width set for image, fixed